.github/workflows/eden.yml

```diff
@@ -0,0 +1,47 @@
+name: Eden
+on:
+  pull_request_review:
+    branches: [master]
+    types: [submitted]
+
+jobs:
+  integration:
+    runs-on: ubuntu-20.04
+    if: ${{ github.event.review.state == 'approved' }}
```

Comment:
I am torn on this. On the one hand, I like the fact that we broke this into two stages: 

1. On PR open, run unit tests, etc.
1. On PR approve, run integration tests

On the other hand, everyone looks at "PR approve" as "good to merge", so it can get confusing. I will try and find time to find a good robot for this.

What might help is if we protected the branch such that tests _must_ pass for the merge to go in. That would also a nice clean "cannot merge anyways, even after PR approval, because tests are running."

Reply:
Don't disagree -- but I think it is OK for now (remember -- we're only starting to move to GH actions -- so a lot will be improved). Or to put it another way -- it is definitely not intrusive enough just yet to not be annoying (one may even say that you should go out of your way to trigger it -- which is sort of by design not to annoy people). This is the same deal as me not disabling CircleCI for now during this transition period.

Now, as for the robot -- I'm all for it. BUT. I really don't want it to be an external one since I'm getting really paranoid about sharing our GH token, but if you can find something that would plug into GH actions -- I'd owe you a 🍺 

Deal?

Reply:
You already owe me several, I believe. :-)